packages/vite/hmr.js

```diff
@@ -64,7 +103,8 @@ async function refresh() {
 		}
 		context.model.off();
 		emptyElement(context.el);
-		let cleanup = await render({ model: context.model, el: context.el });
+		afm.initialize?.({ model: context.model });
+		let cleanup = await afm.render({ model: context.model, el: context.el });
```

Comment:
Both `initialize` and `render` can optionally return a cleanup function. I think we'd want to run those in sequence:

```ts
		let cleanupIntialize = await afm.initialize?.({ model: context.model });
		let cleanupRender = await afm.render({
			model: context.model,
			el: context.el,
		});
		context.cleanup = () => {
			cleanupIntialize?.();
			cleanupRender?.();
		};
```

Reply:
Thanks for the feedback